api/lib/utilities/ccdDataStoreApiPayloads.ts

```diff
@@ -0,0 +1,95 @@
+import {DMDocument} from '../models'
+
+/**
+ * prepareCaseForUpload
+ *
+ * Generic Upload Payload that will be used across all service lines.
+ *
+ * TODO: The TBC's we as a team have asked our service lines to implement.
+ *
+ * @param eventToken
+ * @param eventId
+ * @param dmDocument
+ * @param comments
+ * @return
+ */
+export function prepareCaseForUpload(eventToken, eventId, dmDocument: DMDocument, comments) {
+
+    return {
+        data: {
+            uploadDocuments: [
+                {
+                    value: {
+                        documentName: dmDocument.originalDocumentName,
+                        documentType: 'Letter',
+                        createdBy: dmDocument.createdBy,
+                        createdDate: '2019-01-29',
```

Comment:
Is this data and time hardcoded for a reason?

Reply:
Moved to support folder, as will be needed when a service line implements the case definitions Gareth / we've defined for them.